docsrc/content/abstraction-monad.fsx

```diff
@@ -193,7 +194,7 @@ let getValidPassword : ResultT<_> =
 let askPassword = monad {
     do! lift <| putStrLn "Insert your new password:"
     let! value = getValidPassword
-    do! lift <| putStrLn "Storing in database..."
+    //do! lift <| putStrLn "Storing in database..."
```

Comment:
I'm guessing this caused an error? ..or maybe it was unintentional?

Reply:
Checking locally, I got:

```
   docsrc/content/abstraction-monad.fsx: error(197,8)-(197,53) Type constraint mismatch when applying the default type 'obj' for a type inference variable. The type 'obj' does not support the operator 'Lift' Consider adding further type constraints
```

Adding type constraint didn't help -- I got "Value restriction".
Changing `lift` -> `liftAsync` doesn't help either.

I resolved it by passing unit:

```
let askPassword () = monad {
...
```

Reply:
We should apply your fix to the docs.